arch/arm/src/sama5/sam_can.c

```diff
@@ -1604,7 +1605,7 @@ static void can_interrupt(int irq, void *context, FAR void *arg)
  *      (1 x tCAN).
  *   2. Propagation segment (PROP_SEG):  This part of the bit time is used
  *      to compensate for the physical delay times within the network. It is
- *      twice the sum of the signal�s propagation time on the bus line, the
+ *      twice the sum of the signal�s propagation time on the bus line, the
```

Comment:
@xiaoxiang781216 
unnecessary change?

Reply:
Yes, I revert the change. Please review again.